bedrock/landing/templates/landing/includes/footer.html

```diff
@@ -0,0 +1,31 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+<footer class="mzp-c-footer sem-footer {% if theme_class %}{{ theme_class }}{% endif %}">
+  <div class="mzp-l-content">
+    <nav class="mzp-c-footer-secondary">
+      <div class="mzp-c-footer-legal">
+        <p class="mzp-c-footer-license">
+          Portions of this content are ©1998–2018 by
+          individual mozilla.org contributors.
+          Content available under a
+          <a rel="license" href="https://www.mozilla.org/foundation/licensing/website-content/">Creative Commons license</a>.
+        </p>
+        <ul class="mzp-c-footer-terms">
+          <li>
+            <a href="https://www.mozilla.org/privacy/websites/">Website Privacy Notice</a>
+          </li>
+          <li>
+            <a href="https://www.mozilla.org/privacy/websites/#cookies">Cookies</a>
```

Comment:
At some point the privacy policy was updated and the `#cookies` anchor changed to `#user-choices`.

```suggestion
            <a href="{{ url('privacy.notices.websites') }}#user-choices">Cookies</a>
```